test/integration/a_cart_items_quantity_can_be_updated_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.